k8s-training/terraform.tfvars

```diff
@@ -1,11 +1,11 @@
 # Cloud environment and network
-# parent_id      = "" # The project-id in this context
-# subnet_id      = "" # Use the command "nebius vpc v1alpha1 network list" to see the subnet id
-# ssh_user_name  = "" # Username you want to use to connect to the nodes
-# ssh_public_key = {
-# key  = "put your public ssh key here" OR
-# path = "put path to ssh key here"
-# }
+parent_id      = "" # The project-id in this context
```

Comment:
Empty strings as values will broke tests and could lead con undefined behavior in real usage.